Chatkit/Repositories/States/JoinedRoomsRepositoryState.swift

```diff
@@ -1,10 +1,8 @@
 import Foundation
 
-public extension JoinedRoomsRepository {
-    
     /// An enumeration representing the state of a `JoinedRoomsRepository` which serves live data
     /// retrieved from the Chatkit web service.
-    enum State {
+    public enum JoinedRoomsRepositoryState {
```

Comment:
Looks like this whole class might be pushed out by an extra tab stop.

Reply:
I will fix that.